subsys/settings/src/settings_store.c

```diff
@@ -116,6 +116,11 @@ int settings_delete(const char *name)
 }
 
 int settings_save(void)
+{
+	return settings_save_subtree(NULL);
+}
```

Comment:
nit: I would put this after `settings_save_subtree`, does not matter since both are public but it makes logical sense to use it after the implementation, to me anyway

Reply:
Thanks for the review. I agree about the order, but I tried to keep it consistent with what is already done for `settings_load_subtree` above, or `settings_load_subtree` in `settings.c`.